frodo.js

```diff
@@ -1,20 +1,20 @@
 /**
```

Comment:
SCRIPT-Consider extracting this repeated logic into a separate function to avoid duplication.